lib/rspec/core/memoized_helpers.rb

```diff
@@ -285,6 +285,9 @@ def let(name, &block)
           # We have to pass the block directly to `define_method` to
           # allow it to use method constructs like `super` and `return`.
           raise "#let or #subject called without a block" if block.nil?
+          raise(
+            "#let or #subject called with a reserved name #initialize"
+          ) if "initialize" == name.to_s
```

Comment:
Why `"initialize" == name.to_s` instead of `:initialize == name`?  The symbol form avoids creating an extra string and doing a method call, and is shorter.